features/clipboard-unsanitized-formats.yml

```diff
@@ -1,5 +1,5 @@
-name: Read unsanitized clipboard data
-description: The optional `formats.unsanitized` parameter of the `navigator.clipboard.read()` method reads unsanitized data from the system clipboard.
+name: Read unsanitized HTML clipboard data
```

Comment:
Non-blocking comment: Seems like the spec goes out of its way to make this a list, even though the list only has HTML right now. Not making this specific to HTML future-proofs it to changes.

Reply:
```suggestion
name: Read unsanitized clipboard data
```